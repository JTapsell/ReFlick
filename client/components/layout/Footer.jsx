import React from 'react'

export default function Footer () {
  return (

    <div className="footer">
      <footer> <br/>

        <h4>ABOUT REFLICK</h4>
        <br/>
        <p>A homage and modern remaster of the original open-source flick game, built using React-Redux</p>
        <br/>
        <h4>ABOUT US!</h4>
        <br/>
        <p>Inspired by flickgame.org and developed by Alice, Eve, Azaan, John and Struan.</p>
      </footer>
    </div>
  )
}
