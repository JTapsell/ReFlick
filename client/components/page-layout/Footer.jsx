import React from 'react'

export default function Footer() {
  return (
    <div className="footer">
      <footer>About us</footer>
    </div>
  )
}
