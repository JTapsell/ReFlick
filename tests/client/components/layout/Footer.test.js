import Footer from '../../../../client/components/layout/Footer'
import { shallow } from 'enzyme'
import React from 'react'

test('test runner is working', () => {
  expect(true).toBeTruthy()
})

let wrapper

beforeEach(() => {
  wrapper = shallow(<Footer />)
})
describe('<Footer /> is running', () => {
  it('Renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
  it('Renders coorect text', () => {
    expect(wrapper.find('footer').text()).toContain(' ABOUT REFLICK')
  })
})
