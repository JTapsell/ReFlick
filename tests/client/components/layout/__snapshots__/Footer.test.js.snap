// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> is running Renders correctly 1`] = `ShallowWrapper {}`;
